index.js

```diff
@@ -376,4 +407,41 @@ export default class OneSignal {
 
         RNOneSignal.removeExternalUserId();
     }
+
+    static addTriggers(triggers) {
```

Comment:
Need to add guards around these new methods so it doesn't try to call them for iOS yet.

The other option would be to stub them out in the iOS Objective-C file.